adapters/adapters-commons/src/test/java/uk/gov/justice/raml/common/validator/RequestContentTypeRamlValidatorTest.java

```diff
@@ -36,11 +37,11 @@ public void shouldIgnoreInvalidMediaTypesInNonPOSTActions() throws Exception {
         validator.validate(
                 raml()
                         .with(resource()
-                                .with(action(GET, "application/vnd.structure.dummy.command1+json"))
-                                .with(action(POST, "application/vnd.structure.command.command2+json"))
-                                .with(action(HEAD, "application/vnd.structure.dummy.command3+json"))
-                                .with(action(PUT, "application/vnd.structure.dummy.command4+json"))
-                                .with(action(OPTIONS, "application/vnd.structure.dummy.command5+json"))
+                                .with(httpAction(GET, "application/vnd.structure.dummy.command1+json"))
```

Comment:
Do we want to leave the structure.dummy bits in here and below also, if we are simplifying the paths/names now.